docs/Makefile

```diff
@@ -1,2 +1,2 @@
 all:
-	julia --color=yes make.jl
+	julia --project=. --color=yes make.jl
```

Comment:
I'd be very pleased to know why it's necessary (for my own information)

Reply:
Since we have `Project.toml` in that dir.

`julia --project=.` implies `julia -e 'using Pkg; Pkg.activate(".")'`

Reply:
And you can checkout the doc deployment instructions in `.travis.yml` as well